src/interactive.js

```diff
@@ -0,0 +1,36 @@
+class Interact {
+  constructor(todos) {
+    this.todos = todos;
+  }
+
+  displaylist() {
+    const list = document.querySelector('#todo-data');
+    this.todos.forEach((task) => {
+      const li = document.createElement('li');
+      const text = `<div class="list-container"> <input class='check-input' type='checkbox' value='${task.completed}' aria-label='...'>
+          <p class="todo-text">${task.itemText}</p><a class="del-menu" href="#"><i class="fas fa-ellipsis-v"></i></a></div>`;
+      li.classList.add('list-item');
+      li.innerHTML = text;
+      list.appendChild(li);
+    });
+    localStorage.setItem('todos', JSON.stringify(this.todos));
+    const check = document.querySelectorAll('.check-input');
+    const todos = JSON.parse(localStorage.getItem('todos'));
+    check.forEach((check, index) => {
+      check.addEventListener('change', (e) => {
+        e.preventDefault();
+        const divWrapper = check.parentElement;
+        if (check.checked) {
+          divWrapper.classList.add('checked');
+          todos[index].completed = true;
+        } else {
+          divWrapper.classList.remove('checked');
+          todos[index].completed = false;
+        }
+        localStorage.setItem('todos', JSON.stringify(todos));
```

Comment:
- [ ] After reloading the page, I'm not getting the correct list values. whatever I have marked as complete will again back to the `not completed state`. Please fix this.